apps/src/lib/node/ledger/shell/vote_extensions.rs

```diff
@@ -124,7 +135,7 @@ where
                 .get_current_decision_height(),
             #[cfg(not(feature = "abcipp"))]
             block_height: self.wl_storage.storage.get_last_block_height(),
-            ethereum_events: self.new_ethereum_events(),
+            ethereum_events,
```

Comment:
It seems possible to send a vote extension here when the `ethereum_events` are empty.

Reply:
this is the intended behavior. we always sign sets of ethereum events, even if we find none. this can be changed, but it's out of the scope of this PR.

Reply:
But I think with the retransmission, it will retransmit empty vexts no?

Reply:
oh, I think so yeah. I'll add a check